coredis/commands/core.py

```diff
@@ -2050,7 +2050,9 @@ async def hmset(self, key: KeyT, field_values: Dict[StringT, ValueT]) -> bool:
         readonly=True,
         group=CommandGroup.HASH,
     )
-    async def hmget(self, key: KeyT, fields: Parameters[StringT]) -> Tuple[AnyStr, ...]:
+    async def hmget(
+        self, key: KeyT, fields: Parameters[StringT]
+    ) -> Tuple[Optional[AnyStr], ...]:
```

Comment:
Minor nit, could you please also update the `TupleCallback` specialization to use `Optional[AnyStr]`?

Reply:
done

Reply:
:bow: